aws-lc-sys/builder/cc_builder/aarch64_apple_darwin.rs

```diff
@@ -0,0 +1,317 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+// Wed Apr  3 12:53:57 UTC 2024
+
+use crate::cc_builder::Library;
+
+pub(super) const CRYPTO_LIBRARY: Library = Library {
+    name: "crypto",
+    flags: &[],
```

Comment:
Are these supposed to have values? or is it for future use?

Reply:
I thought I might end up needing to specify flags that could vary by platform, but that's not been needed yet. I can remove the attribute if needed.

Reply:
I think it's fine to keep. I just wasn't sure if there were values supposed to be here on at least one platform. Maybe a comment for now?